Demo/Fullscreen/FullscreenGallery/FullscreenGalleryDemoViewController.swift

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright (c) 2019 FINN AS. All rights reserved.
+//
+
+class FullscreenGalleryDemoViewController: DemoViewController<FullscreenGalleryDemoView> {
```

Comment:
> Is it because of limitations of the Demo project where we work mostly with views?

This was indeed the reasoning behind this, but moving this into the custom view-controller sounds like a much better solution 🙂 👍